### ISSUE TITLE
DST Timex.Timezone.total_offset/1 error

### ISSUE THREAD

**Issue by almassapargali** [bitwalker/timex#485]
### Steps to reproduce

So I have this line, where timezone is `"America/New_York"`
```
    Timex.Timezone.get(timezone) |> Timex.Timezone.total_offset
```

Right now it's working correctly again and returning `-18000`, but 04.11.18 2:00 UTC it were failing with: 

```
** (exit) an exception was raised:
    ** (FunctionClauseError) no function clause matching in Timex.Timezone.total_offset/1
        (timex) lib/timezone/timezone.ex:283: Timex.Timezone.total_offset(#<Ambiguous(#<TimezoneInfo(America/New_York - EDT (-04:00:00))> ~ #<TimezoneInfo(America/New_York - EST (-05:00:00))>)>)
```

That's just around DST time switch, and I guess it might be related to it.

**Comment by baldmountain** [comment]
@almassapargali  This is fixed if you update to the latest version 3.4.2.

**Comment by almassapargali** [comment]
Ok, thanks!
